src/app.ts

```
@@ -139,982 +138,967 @@
 };
 
-const fedifyKv = await KnexKvStore.create(client, 'key_value');
-
 let queue: GCloudPubSubPushMessageQueue | undefined;
+let fedify: ReturnType<typeof createFederation<ContextData>>;
+type FedifyRequestContext = ReturnType<typeof createFederation<ContextData>>;
+type FedifyContext = Context<ContextData>;
+type HonoContextVariables = ContextData;
+type AppContext = HonoContext<{ Variables: HonoContextVariables }>
+let db: KvStore;
+export { fedify, FedifyRequestContext, FedifyContext, db, HonoContextVariables, AppContext };
```

Biome (1.9.4)

[error] Some exports are only types.

This export is a type.

This export is a type.

This export is a type.

This export is a type.

Using export type allows compilers to safely drop exports of types without looking for their definition.
Safe fix: Add inline type keywords.


(lint/style/useExportType)